docs/5.0/admin-guide.md

```diff
@@ -173,7 +173,8 @@ teleport:
   auth_token: xxxx-token-xxxx
 
   # Optional CA pin of the auth server. This enables more secure way of adding new
-  # nodes to a cluster. See "Adding Nodes" section above.
+  # nodes to a cluster. See "Adding Nodes to the Cluster"
+  # (https://goteleport.com/teleport/docs/admin-guide/#adding-nodes-to-the-cluster).
```

Comment:
All of these links should be relative rather than absolute as they're on the same page.

See `["Adding Nodes to the Cluster"](#adding-nodes-to-the-cluster)`